src/main/java/core/basesyntax/service/FileWriterService.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface FileWriterService {
+    void write(String data, String toFileName, String path);
```

Comment:
The method signature for 'write' could be more descriptive. Consider renaming 'data' to 'content' to clarify that it represents the content to be written to the file. Also, 'toFileName' and 'path' could be combined into a single parameter 'filePath' to specify the file's location more clearly. The revised method signature could be 'void write(String content, String filePath)'.

Reply:
++